app/screens/channel/index.tsx

```diff
@@ -29,6 +30,7 @@ const enhanced = withObservables([], ({database, serverUrl}: EnhanceProps) => {
     const currentUserId = observeCurrentUserId(database);
     const hasGMasDMFeature = observeHasGMasDMFeature(database);
     const isBookmarksEnabled = observeConfigBooleanValue(database, 'FeatureFlagChannelBookmarks');
+    const isCRTEnabled = observeIsCRTEnabled(database);
```

Comment:
do we need this here? I think that can be moved to the index file of `SchedulePostIndicator`

Reply:
Sure